example/sampleProp.conf

```diff
@@ -7,6 +7,33 @@ GraphiteIP = "Graphite-IP"
 GraphitePort = None
 Device1Name = "Device-Name"
 
+#MQTT Configurations
```

Comment:
This file is growing too big and most of the parameters can be irrelevant to other examples. We can create sections like [AWS], [VROPS] etc or can make altogether new config file say, aws_sampleProp.conf if nothing is shared.